addons/message_update_v2.13/manifest.json

```
@@ -0,0 +1,74 @@
+{
+  "api_version": "0.1",
+  "id": "message_update_v2.13",
+  "name": "Update to Mozilla VPN 2.13",
+  "type": "message",
+  "conditions": {
+    "max_client_version": "2.12.9"
+  },
+  "javascript": {
+    "enable": "enable.js"
+  },
+  "message": {
+    "date": 1671066000,
+    "id": "message_update_v2.13",
+    "title": "Update to Mozilla VPN 2.13",
+    "subtitle": "We’ve released an updated version of Mozilla VPN! Update to the latest version to get the newest features and bug fixes: ",
```

```suggestion
    "subtitle": "We’ve released an updated version of Mozilla VPN! Update to the latest version to get the newest features and bug fixes:",
```

No need for a trailing whitespace.